### DISCUSSION TITLE
Critical Cartography and Participatory Mapping

### DISCUSSION THREAD

**Discussion by avaitt** [albertkun/21S-ASIAAM-191A#86] [Past Thinking Caps]
Participatory mapping and critical cartography allow us to create ethical visualizations that can be used for the greater good. However, just with any visualization, participatory mapping and critical cartography can skew information. Some drawbacks including mislabeling data, as well as illustrating a false narrative or a narrative that the author might want to push forwards. Looking at discussion and lecture narratives can be empowered and many cultures can be erased through cartography. Participatory mapping can help revive cultures and words that might have been erased or phased out. By bringing to light the stories and words of the marginalized participatory mapping and critical cartography can be used as a force of good.

**Comment by latishasumardy** [comment]
I agree! Its very easy for groups to push their own narratives, which I feel is displayed prominently when viewing news reports of the same events from different news channels. I also agree with your point on how many cultures have been essentially erased from history due to cartography, as this is especially seen when you see how the indigenous people groups in America are not even referenced in many maps of America we see today. I believe that these tools will allow underrepresented groups to tell their own stories and get their messages out to the world.

**Comment by annyrviloria** [comment]
So true, I think it's humbling to think about how one discipline/tool/method that has had some really deleterious effect on people's lives can also be used as a force of good. I majored in anthropology when I was in undergrad and learning about the ways in which anthropology has been used to advance racist ideologies (think about phrenology, just as an example) was in some ways challenging and discouraging. Yet, coming into anthropology with the desire to uplift the voices of my community, and to take back the power that written history had taken away from us was deeply motivating and grounding. I see a parallel with the ways cartography has been used for the erasure of indigenous peoples and at the same time, the immense power it holds to be deployed for the benefit of those same communities if done right.
